src/core/core.js

```diff
@@ -442,7 +466,8 @@ Crafty.fn = Crafty.prototype = {
             }
         }
         delete this.__c[id];
-
+        // update map from component to (entityId -> entity)
```

Comment:
I find this description a little confusing.  I'd probably call it a map from component -> (hashset of entity ids).